dependencies.gradle

```diff
@@ -30,6 +30,7 @@ ext {
         sheetMenu              : '1.3.3',
         aVLoadingIndicatorView : '2.1.3',
         flexbox                : '0.3.2',
+        customTabs             : '27.0.2',
```

Comment:
You don't need to specify it here. CustomTabs uses Support Library, so remove it and use 
https://github.com/WideChat/Rocket.Chat.Android/blob/18aa1d623afaf721b2108f694ef720f4db4f5faf/dependencies.gradle#L12